### ISSUE TITLE
Provide open source version snapshots

### ISSUE THREAD

**Issue by strindberg** [jOOQ/jOOQ#14989]
### Use case

It seems that the snapshot builds of the open source version available here (https://www.jooq.org/download/versions#oss) can only be downloaded if you have a license for one of the commercial versions; the links are only clickable once one has supplied license info. Is this as intended? I imagine it would sometimes be nice to get hold of these open source snapshots even without a commercial license.

### jOOQ Version

3.19.0-SNAPSHOT

### Database product and version

Any

**Comment by lukaseder** [comment]
Release numbers are cheap. If a bug is fixed, then the tests pass.

Occasionally, *your* bug may still not be fixed yet, because the bug understanding is different : But that's very rare. In case it isn't fixed, we can re-iterate.

When I decided to offer snapshot builds only to paying customers, I weighed the benefits of having an additional value add for paying customers only (early feature access on tested builds) vs the occasional faster feedback cycle for a bugfix

Given how rarely bugs are closed prematurely, I think the tradeoff is favourable.
